roles/test_report_send/tasks/metadata.detect/dci.yml

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# metadata.detect/dci.yml tasks file for redhatci.ocp.test_report_send
+
+- name: Set is_ci attribute
+  ansible.builtin.set_fact:
+    trs_ci: "{{ trs_ci | default({}) | combine({'is_ci': true}) }}"
+  when:
+    - lookup("env","CI") | lower == "true"
+    - lookup("env","DCI_CS_URL") is defined
```

Comment:
metadata.detect/dci.yml is called when `DCI_CS_URL` is defined in `ci.detect.yml`, no need to check here.